install_config/installing-pipelines.adoc

```diff
@@ -44,13 +44,13 @@ include::modules/op-performance-tuning-using-tektonconfig-cr.adoc[leveloffset=+1
 
 * You can learn more about installing Operators on {OCP} in the link:https://docs.openshift.com/container-platform/latest/operators/admin/olm-adding-operators-to-cluster.html#olm-adding-operators-to-a-cluster[adding Operators to a cluster] section.
 
-* To install {tekton-chains} using the {pipelines-title} Operator, see xref:../pipelines/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc#using-tekton-chains-for-openshift-pipelines-supply-chain-security[Using {tekton-chains} for {pipelines-title} supply chain security].
+* To install {tekton-chains} using the {pipelines-title} Operator, see xref:../secure/using-tekton-chains-for-openshift-pipelines-supply-chain-security.adoc#using-tekton-chains-for-openshift-pipelines-supply-chain-security[Using {tekton-chains} for {pipelines-title} supply chain security].
```

Comment:
Comment fro L13
@mramendi Why is the prereqs discrete here? I do not see such usage in the OCP docs or a mention such as this for prereqs in the OCP docs guidelines anywhere. Please refer to https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#anchoring-prerequisites-additional-resources-and-next-steps-titles-in-assemblies